### ISSUE TITLE
ColorLoss with file recorder

### ISSUE THREAD

**Issue by TKooijmans** [CasparCG/Server#488]
When a file is recorded with any codec the resulting clip has colorloss.
This is clearly visible with Colorbar recording.

It seems that colorspace 701 is not applied before recording. (tested with HD 1080i50)

**Comment by TKooijmans** [comment]
This happens with all types of codecs, like mpg, h264 , MXF and raw or quicktime
here are some examples
ADD 1 FILE Testcap.mp4 -x264opts colorprim=bt709:transfer=bt709:colormatrix=bt709:fullrange=off 
Result, color loss

ADD 1 FILE "Testcap.mp4" -x264opts  colorprim=bt709:transfer=bt709:colormatrix=bt709:fullrange=on 
Result, color is fine but contrast/blacklevel is changed

ADD 1 FILE Testcap.mp4 -x264opts colorprim=bt601:transfer=bt601:colormatrix=bt601:fullrange=off
Result Error does not capture

Test setup,
play a colorbar on channel 1-1
record that channel
compare the two...

**Comment by TKooijmans** [comment]
That's great!
is this now fixed on the latest 2.1 build or on 2.07 build too?

**Comment by hummelstrand** [comment]
@TKooijmans This was fixed in this commit https://github.com/CasparCG/Server/commit/3ad74adfa16def07a378b63c832cf4471a1819f0 in Server 2.1. 
Server 2.0.x is not being developed anymore and hasn't had any code changes for over a year.

**Comment by TKooijmans** [comment]
I just tested this with the build of 28-11-2016. (3258)
The colorloss is gone! but now other options are ignored.
Also all h264 files are recorded in YUV444 format and unlimited bitrate
and mxf files are always recorded in yuv420, no HDcam anymore.

These option settings I used for 2.07 are ignored in 2.1
ADD 1 Recfile.mxf -target xdcamhd422 -vtag xd5c -pix_fmt yuv422p -b 50000k 
should record a HDcam50 file but is now yuv420 and bitrate is ignored too

ADD 1 FILE testH264.mp4 -preset ultrafast -b 12000000 causes an error

ADD 1 FILE testH264.mp4
is always recorded in yuv444 intra  120- 150 mbit datarate
